sormas-e2e-performance-tests/README.md

```diff
@@ -0,0 +1,35 @@
+# Table of contents
+
+* [General info](#general-info)
+* [Setup](#setup)
+* [Test execution](#test-execution)
+* [Reporting](#reporting)
+
+## General info
+
+This project aims to help identify performance issues on the Sormas rest api's.
+
+## Setup
+* Install Java JDK 11 on your local machine
+* Install apache Jmeter
+* Clone the project and open the *sormas-e2e-performance-tests* subproject
+
+
+## Test execution
+
+Non-GUI execution of the performance tests is recommended:
+
+> Execution from a terminal window:
+* open the console and navigate to the Jmeter bin folder
+* type the following in the command line:  jmeter -n -t 'PathToTheJMXFile'\SormasPoc.jmx 
+  -l 'PathWhereTheResultsWillBeSaved'\Results.csv -e -o 'PathWhereTheHtmlReportWillBeSaved'\reports
```

Comment:
Would be good to add number of threads info too just like the one mentioned in the example below.

Reply:
It is just an example from my local path(I usually create folders with number of threads for results and report). But as we discussed, I will simplify the example for more clarity. Thank you